M2/Macaulay2/e/ExponentVector.hpp

```diff
@@ -272,13 +272,13 @@ class ExponentVector
 };
 
 // Legacy specialization
-using ntuple = ExponentVector<int, true>;
-typedef ntuple::Exponents exponents_t;
-typedef ntuple::ConstExponents const_exponents;
+using exponents = ExponentVector<int, true>;
+typedef exponents::Exponents exponents_t;
```

Comment:
There wasn't a particular reason, is one preferred over the other? I'm only using `using` in the first line because that's the only way to alias a template specialization.

Reply:
It seems like the `using` form supercedes the older `typedef` form completely, as far as I can tell.  Basically the syntax is clearer, and it handles more cases (such as templates, as you are using here).  But we can change it later too.